deployments/ansible/roles/collector/tasks/otel_win_install.yml

```diff
@@ -1,42 +1,18 @@
 ---
 
-- name: Get splunk-otel-collector latest version
```

Comment:
Deleted lines from this file are simply being moved to the `win_download_remote_msi,yml` task in the same directory.